app-text/calibre/calibre-3.29.0.recipe

```diff
@@ -0,0 +1,174 @@
+SUMMARY="Ebook management application"
+DESCRIPTION="Calibre is an e-book manager. It can view, convert, edit and \
+catalog e-books in all of the major e-book formats. It can also talk to \
+e-book reader devices. It can go out to the internet and fetch metadata for \
+your books. It can download newspapers and convert them into e-books for \
+convenient reading. It is cross platform, running on Linux, Windows and macOS."
+HOMEPAGE="https://calibre-ebook.com/"
+COPYRIGHT="2008-2017 Kovid Goyal"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/kovidgoyal/calibre/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="0eddc3fef236edefa01d7a576e63168b725f5fc872389250eb937280a5018c23"
+
```

Comment:
```diff
 CHECKSUM_SHA256="0eddc3fef236edefa01d7a576e63168b725f5fc872389250eb937280a5018c23"
+SOURCE_FILENAME="calibre-$portVersion.tar.gz"
```

FWIW, I noticed that there is also a `tar.xz` archive, and I was going to suggest this change:
```diff
-SOURCE_URI="https://github.com/kovidgoyal/calibre/archive/v$portVersion.tar.gz"
+SOURCE_URI="https://github.com/kovidgoyal/calibre/releases/download/v$portVersion/calibre-$portVersion.tar.xz
-CHECKSUM_SHA256="0eddc3fef236edefa01d7a576e63168b725f5fc872389250eb937280a5018c23"
+CHECKSUM_SHA256="d0577e507e4e7244ce3f3f06acc3b0f3ffb87d26e32d79bfb520872251dc49e4"
```
But this tarball is 38 MiB, while the `tar.gz` archive is 21 MiB.
(The `tar.xz` archive has localized manual pages and documentation.)
(Any opinions on whether or not we should include the localized man pages?)